main/src/main/scala/sbt/internal/CommandExchange.scala

```diff
@@ -168,34 +168,20 @@ private[sbt] final class CommandExchange {
   }
 
   def publishEvent[A: JsonFormat](event: A): Unit = {
-    val broadcastStringMessage = true
```

Comment:
I was thinking about potentially turning this into a client specific setting.

Reply:
I see.